tools/ocean/argo_getdata.xml

```diff
@@ -0,0 +1,349 @@
+                <option value="space_time">🗺 For a space/time domain</option>
+                <option value="floats">🤖 For one or more floats</option>
+                <option value="profiles">⚓ For one or more profiles</option>
+            </param>
+            <when value="space_time">
+                <param name="cardinal_1" type="float" min="-90" max="90" value="0" label="Input longitude min (+east/-west):"/>
+                <param name="cardinal_2" type="float" min="-90" max="90" value="0" label="Input longitude max (+east/-west):"/>
+                <param name="cardinal_3" type="float" min="-90" max="90" value="0" label="Input latitude min (+north/-south):"/>
+                <param name="cardinal_4" type="float" min="-90" max="90" value="0" label="Input latitude max (+north/-south):"/>
+                <param name="pressure_1" type="float" value="0" label="Input pressure min (db)"/>
+                <param name="pressure_2" type="float" value="0" label="Input pressure max (db)"/>
+                <param name="date_1" type="text" optional="true" label="Input starting date" value="yyyy-mm"/>
+                <param name="date_2" type="text" optional="true" label="Input ending date" value="yyyy-mm"/>    
+            </when>
+            <when value="floats">
+                <param name="wmo" type="text" label="Input WMO number" help="if you want to enter multiple ones just separate them with a comma (example : 6902746, 6902755)"/>
+            </when>
+            <when value="profiles">
+                <param name="wmo" type="integer" value="0" label="Input WMO number" />
+                <param name="profile" type="text" multiple="true" value="0" label="Input profile cycle number" help="if you want to enter multiple ones just separate them with a comma (example : 3,12)"/>
+            </when>
+        </conditional>
+        <!--<conditional name="sources">
+            <param name="data_sources" type="select" label="From where do you want to get your data ?">
+                <option value="erddap">⭐ the Ifremer erddap server </option>
+                <option value="gdac">🌐 an Argo GDAC server or any other GDAC-compliant local folder.</option>
+                <option value="argovis">👁 the Argovis server.</option>
+            </param>
+            <when value="erddap"/>
+            <when value="gdac">
+                <param name="ftp" type="select" label="Input ftp server url">
+                    <option value="ftp://ftp.ifremer.fr/ifremer/argo">France Coriolis</option>
+                    <option value="ftp://usgodae.org/pub/outgoing/argo">US GODAE</option>
+                </param>
+            </when>
+            <when value="argovis"/>
+        </conditional>-->
+        <conditional name="dataset">
+            <param name="data_dataset" type="select" label="Which kind of dataset do you want ?" display="radio">
```

Comment:
Do you need display=radio here?

radio is more a select and not a conditional imho